src/test/java/teammates/test/cases/ui/browsertests/StudentProfilePageUiTest.java

```diff
@@ -284,7 +284,7 @@ private StudentProfilePage getProfilePageForStudent(String studentId) {
     }
 
     @AfterClass
-    public void testAfter() {
+    public static void classTearDown() throws Exception {
```

Comment:
Out of curiosity, is this method static as required by Travis, or just to make it similar to other UI tests?